services/miner/cpuminer.go

```diff
@@ -344,7 +344,7 @@ func (m *CPUMiner) solveBlock(msgBlock *types.Block, ticker *time.Ticker, quit c
 		instance.SetParams(m.params.PowConfig)
 		hashesCompleted += 2
 		header.Pow = instance
-		if instance.FindSolver(header.BlockData(), header.BlockHash(), header.Difficulty) {
+		if header.Pow.FindSolver(header.BlockData(), header.BlockHash(), header.Difficulty) {
```

Comment:
- I don't think the change results any different than before. 
-  we might have to rethink the role/relationship between  `header` and `instance`, is it necessary of the field `header.Pow` and `header.Pow = instance` ? for me, `header.Pow` looks useless in the method.